CHANGES.md

```diff
@@ -4,7 +4,9 @@ Change Log
 ### 1.47 - 2018-07-02
 
 ##### Breaking Changes :mega:
-* glTF 2.0 models corrected to face +Z forwards per specification.  Internally Cesium uses +X as forward, so a new +Z to +X rotation was added for 2.0 models only.  [#6632](https://github.com/AnalyticalGraphicsInc/cesium/pull/6632)
+* glTF 2.0 models corrected to face +Z forwards per specification.  Internally Cesium uses +X as forward, so a new +Z to +X rotation was added for 2.0 models only. To fix models that are oriented incorrectly after this change:
+    * If the model faces +X forwards update the glTF to face +Z forwards. This can be done by loading the glTF in a model editor and applying a -90 degree rotation about the Y-axis. Alternatively, add a new root node to the glTF node hierarchy whose `matrix` is `[0,0,1,0,0,1,0,0,-1,0,0,0,0,0,0,1]`.
```

Comment:
Some modeling programs might have a different up/forward convention, so maybe it's better to specify something like `a 90 degree clockwise rotation about the Up-axis`.